Example/IntegrationTests/Push/PushTests.swift

```diff
@@ -21,10 +23,19 @@ final class PushTests: XCTestCase {
 
     var pairingStorage: PairingStorage!
 
+    let pk = try! EthereumPrivateKey()
```

Comment:
sometimes we re running those tests against prod cast and prod gm dapp. Can we use some fixed entropy here for deriving the private key? to not pollute their DBs

Reply:
Why we can pollute db's? I think there will be same amount of relay messages in both cases (random and derived private key)